packages/components/src/components/quickstart/QuickstartLayout.vue

```diff
@@ -122,7 +122,7 @@ body {
 
 .qs-content {
   margin-bottom: 12px;
-  background: $color-tile-background;
+  background: $gray2;
```

Comment:
The `$color-*` variables [defined here](https://github.com/applandinc/appmap-js/blob/main/packages/components/src/scss/_variables.scss#L114-L128) were being used to allow the user's VSCode theme to influence the styling used in these components. It's okay if we're moving away from that, but I want to make sure this change is made purposefully.